README.md

```diff
@@ -2,7 +2,38 @@
 [![Build Status](https://travis-ci.org/hmcts/job-scheduler.svg?branch=master)](https://travis-ci.org/hmcts/job-scheduler)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/db1d536343474c40967ab9b236044e1d)](https://www.codacy.com/app/HMCTS/job-scheduler)
 
+The job scheduler micro service allows other services to schedule http actions in the future.
```

Comment:
Worth extending this description by a couple of sentences, so that members of other teams could get a better feel of what this tool is about.